frontend/src/components/checkout/place-order/order-itmes/ItemsList.tsx

```diff
@@ -30,44 +29,57 @@ export default function ItemsList() {
                     price,
                     productLimit,
                     countInStock,
-                }: product) => (
-                    <Card sx={ cardContainer } key={ productId }>
-                        <Card>
-                            <img style={ itemImage } src={ image } alt="items-img" />
-                        </Card>
-                        <Typography>{ productId }</Typography>
-                        <Typography>{ price }$</Typography>
-                        <Button
-                            onClick={ handleQuantityDecrement(productId) }
-                            disabled={ quantity === 1 }
-                        >
-                            <RemoveIcon />
-                        </Button>
-                        <Typography>{ quantity }</Typography>
+                }: product) => {
+
+                    const disabledButtonLogic =
+                        quantity === productLimit ||
+                        quantity === 6 ||
+                        quantity >= countInStock;
```

Comment:
Is it possible to assign variable names to these conditions?
what does this 6 mean?
what do you mean by countInStock?